### ISSUE TITLE
Are Accounts/Actors and Statuses URIs intentionally different?

### ISSUE THREAD

**Issue by diligiant** [davecheney/pub#26]
I just noticed that the first follow this template `https://%s/u/%s` ([here](https://github.com/davecheney/pub/blob/75df84eb28ebc9a36295715e83c5495f0917d668/models/status.go#L243)) whereas the second => `https://%s/users/%s/%d` ([here](https://github.com/davecheney/pub/blob/75df84eb28ebc9a36295715e83c5495f0917d668/models/account.go#LL121C31-L121C46)) and I was wondering if this was intentional or just an oversight when you decided to use `/u/` instead of `/users/`? Some code out there might want to infer the account URI from a status'… ;)

**Comment by davecheney** [comment]
Hmm, yes this is probably a bug. Keeping the activitypub representation of an actor or status separate from its mastodon version always confuses me

I like the way that wilderbeast does it, /ap/o for objects, /ap/u for actors.

**Comment by diligiant** [comment]
Indeed, I like their `/ap/users` and `/ap/o` prefixes as it very explicitly (different root, different sub-root lengths) breaks bad players. I'm willing to try if you're game.

**Comment by diligiant** [comment]
I'm seeing this on some of them but I don't think this happens at the creation of the note.
```
INSERT INTO statuses (…uri…) VALUES (…"https://mastodon.social/users/davew/statuses/110334005148072062"…)
```
